tests/api/test_security.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import os
+
+
+class Testsecurity(object):
+
+    def test_main_page(self, flask_api_url):
+        response = requests.get(flask_api_url + '/')
+        assert response.status_code == 200
+
+    def test_login_fake_user(self, flask_api_url):
+        response = requests.post(flask_api_url + '/api/users/login',
+                                 data=dict(email='user1', password='password'))
+        assert response.reason == 'FORBIDDEN'
+        assert response.status_code == 403
+
+    def test_login_real_user(self, flask_api_url):
+        response = requests.post(flask_api_url + '/api/users/login',
+                                 data=dict(email='admin@knex.com', password="admin"))
+        assert response.reason == 'OK'
+        assert response.status_code == 200
+
+    def test_login_real_user_wrong_psswd(self, flask_api_url):
+        response = requests.post(flask_api_url + '/api/users/login',
+                                 data=dict(email='admin', password='a'))
+        assert response.reason == 'FORBIDDEN'
+        assert response.status_code == 403
+
+    def test_logout(self, flask_api_url):
+        response = requests.get(flask_api_url + '/api/users/login',
+                                data=dict(email='user@knex.com', password="user"))
+        response = requests.get(flask_api_url + '/api/users/logout')
+        assert response.reason == 'OK'
+        assert response.status_code == 200
+
+    def test_access_login_required_logged(self, pytestconfig, flask_api_url):
```

Comment:
this test shouldn't even work, you don't login anywhere. why does this run with assert response.status_code == 200 when it should return 403?